fw/application/src/i18n/messages.c

```diff
@@ -0,0 +1,145 @@
+#include "messages.h"
+#include <stddef.h>
+
+static enum message_languages_t current_language = MESSAGE_LANG_ZH_CN;
+const enum message_languages_t default_language = MESSAGE_LANG_ZH_CN;
+
+const char* get_message_in_language(enum messages_t message_id, enum message_languages_t language_id) {
+    switch(language_id) {
+                case MESSAGE_LANG_ZH_CN:
```

Comment:
please put the message to different c files with an array , for example:

message.h
```
enum message_id {
    MESSAGE_ID_AMIIBO_EMULATOR,
    MESSAGE_ID_BLE,
    MESSAGE_ID_SETTINGS,
    MESSAGE_ID_MAX
};
```
message.c
```
const char* get_message(message_id  id, message_languages_t lang){
   //TODO boundry check...
     if(lang == MESSAGE_LANG_ZH_CN) {
           return lang_zh_cn[id];
       } else {
           return  lang_en_us[id];
       }
}
```

lang_en_us.c
```
const char * lang_en_us[MESSAGE_ID_MAX] = {
    [MESSAGE_ID_AMIIBO_EMULATOR] = "Amiibo Emulator", 
    [MESSAGE_ID_SETTINGS] = "Settings"
};
```

lang_zh_cn.c
```
const char * lang_zh_cn[MESSAGE_ID_MAX] = {
    [MESSAGE_ID_AMIIBO_EMULATOR] = "Amiibo模拟器", 
    [MESSAGE_ID_SETTINGS] = "设置"
};
```